Tests/LiveKitTests/Support/Room.swift

```diff
@@ -182,6 +189,18 @@ extension Room {
     }
 }
 
+extension Track.Source: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .unknown: return "unknown"
+        case .camera: return "camera"
+        case .microphone: return "microphone"
+        case .screenShareVideo: return "screenShareVideo"
+        case .screenShareAudio: return "screenShareAudio"
+        }
+    }
+}
```

Comment:
I think there was a dedicated file for `CustomStringConvertible`s maybe move it there 😀